articles/anomaly-detection/concepts/breached-passwords.md

```diff
@@ -9,11 +9,17 @@ useCase: customize-anomaly-detection
 v2: true
 ---
 
-# Breached Password Security
+# Breached Password Detection
 
-When a user receives an email requesting that they change their password immediately, it is because their account could be the victim of a security breach. This may be the result of a compromise by a third-party application that experienced a security breach. The breach may not have happened to this account, but based on available data, the user's credentials may have been released. Since many people reuse passwords, the request to change passwords is a precaution to make sure the user stays protected. 
+Every day, malicious hackers penetrate websites and applications, exposing thousands of emails and passwords. Because it's common for users to use the same password to login to multiples sites, this poses a problem, not only for the hacked system but to any application that shares those breached passwords.
 
-Users may also want to change their password at any other sites where they suspect they used a shared password. 
+Auth0 tracks large security breaches that are happening on major third-party sites to help secure your users and system. By [enabling breached password detection](/anomaly-detection/guides/set-anomaly-detection-preferences), when a trigger occurs, your users can be notified and/or blocked from logging in if we suspect their credentials were part of a published security breach. 
+
+You can configure the **URL Lifetime** and **Redirect To** values in the Dashboard by going to [Emails > Templates > Change Password Template](${manage_url}/#/emails).
+
+Breached password detection works when logging in using the Resource Owner Password Grant (ROPG) and when using custom databases with your tenants.
```

Comment:
Does it only work with these two? I would assume that it also works with other technologies, such as an Authz Code Flow (in Universal Login) - may be worth to note "[...] only works when [..]" or "[...] also works when [...]"

Reply:
Changed to "also works..."